Manual.md

```diff
@@ -604,7 +604,9 @@ patches to the package sources during `do_patch()`. Patches are stored in
 `srcpkgs/<pkgname>/patches` and must be in `-p1` format. By default set to `-Np1`.
 
 - `disable_parallel_build` If set the package won't be built in parallel
-and `XBPS_MAKEJOBS` has no effect.
+and `XBPS_MAKEJOBS` has no effect. If a package does not work well with `XBPS_MAKEJOBS`
+but still has a mechanism to build in parallel, set `disable_parallel_build` and
+use `XBPS_ORIG_MAKEJOBS` instead.
```

Comment:
> "use `XBPS_ORIG_MAKEJOBS`

This is a bit ambiguous regarding to where from it comes. What about

> and `XBPS_MAKEJOBS` is reset to 1.

> in template read value of `XBPS_ORIG_MAKEJOBS`, which is what `XBPS_MAKEJOBS` was reset from?

Reply:
Updated. How's it look now?